doc/lib.rst

```diff
@@ -123,6 +120,12 @@ Collections
 * `sets <sets.html>`_
   Nim hash and bit set support.
 
+* `std/setutils <setutils.html>`_
+  This module adds functionality for the built-in ``set`` type.
+
+* `std/enumutils <enumutils.html>`_
+  This module adds functionality for the built-in ``enum`` type.
+
```

Comment:
What's the reasoning behind moving this under the "Collections" section? Since both of these modules add utilities/algorithms for existing collection types (rather than new collection types, like the other modules there), I think they better fit under the "Algorithms" section, especially since `sequtils` also is there.

Also, as far as I can tell, the modules should be sorted by their name, ignoring the `std/` prefix, so `std/enumutils` should be higher up either way.

Reply:
> Since both of these modules add utilities/algorithms for existing collection types (rather than new collection types, like the other modules there), I think they better fit under the "Algorithms" section, especially since sequtils also is there.

good points, I've followed your recommendation

> ignoring the std/

IMO after https://github.com/nim-lang/Nim/pull/16999 we should just remove those std/ prefixes from this list (historical artifact that some don't have it, and the module page would have all the details), but can be discussed separately